pkg/apis/node/validation/validation.go

```diff
@@ -41,3 +49,10 @@ func ValidateRuntimeClassUpdate(new, old *node.RuntimeClass) field.ErrorList {
 
 	return allErrs
 }
+
+func validateOverhead(overhead *node.Overhead, fldPath *field.Path) field.ErrorList {
+	resourceReq := core.ResourceRequirements{
```

Comment:
Short comment on the trickiness here would be nice.

Reply:
addressed